reflection/serverreflection.go

```diff
@@ -40,9 +40,12 @@ import (
 	"bytes"
 	"compress/gzip"
 	"fmt"
+	"google.golang.org/protobuf/reflect/protodesc"
```

Comment:
The files need to be formatted. Run goimports/gofmt/golint.
Run `VET_SKIP_PROTO=true ./vet.sh` to verify.